documentation/ipi-install/modules/ipi-install-bmc-addressing.adoc

```diff
@@ -3,7 +3,6 @@
 // * installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
 
 [id="ipi-install-bmc-addressing_{context}"]
-[[bmcaddressing]]
```

Comment:
This reference is also used in install-config-yaml for each release, if we remove it, we'll need to update all the other ones

Reply:
I've made a test by renaming them on my own, but the _context is not expanded when used in the install-config-yaml$RELEASE so I would leave this anchor here and work in another PR on renaming/removing so that it still works

Reply:
Raised https://github.com/openshift-kni/baremetal-deploy/issues/589 on this topic

Reply:
Solved with the new commit I made